### DISCUSSION TITLE
option to disable underline sql errors ?

### DISCUSSION THREAD

**Discussion by emanueol** [dbeaver/dbeaver#17809] [Q&A]
Hi where can i find option to disable underline sql errors ?
![image](https://user-images.githubusercontent.com/8422412/193866245-baa02f1b-3f55-4562-ac34-af13f05ed7bc.png)

thanks

**Comment by LonwoLonwo** [comment]
![2022-10-06 11_43_39-DBeaver 22 2 1 - _DBeaver Sample Database (SQLite)_ Script-35](https://user-images.githubusercontent.com/45152336/194265644-10fca115-42d9-4863-bb8b-4a54f6d3aec7.png)

Hello, Yoy can find this disable this setting in the Prefernces -> Editors -> SQL Editor -> Code Editor -> "Show problem markers"
